.github/workflows/deploy.yml

```diff
@@ -23,11 +23,11 @@ jobs:
           OS_BOTIFY_APP_ID: ${{ secrets.OS_BOTIFY_APP_ID }}
           OS_BOTIFY_PRIVATE_KEY: ${{ secrets.OS_BOTIFY_PRIVATE_KEY }}
 
-      - name: Tag version
-        run: git tag "$(npm run print-version --silent)"
+      - name: Get current app version
+        run: echo "STAGING_VERSION=$(npm run print-version --silent)" >> "$GITHUB_ENV"
 
-      - name: 🚀 Push tags to trigger staging deploy 🚀
-        run: git push --tags
+      - name: 🚀 Create prerelease to trigger staging deploy 🚀
+        run: gh release create ${{ env.STAGING_VERSION }} --title ${{ env.STAGING_VERSION }} --generate-notes --prerelease
```

Comment:
Do we need to create and push the tag before we can create a release from it? Maybe `gh release create` will automatically make the tag, but we definitely need the tag ref to exist in our git history

Reply:
It will create the tag if it does not exist already:

> [If a matching git tag does not yet exist, one will automatically get created from the latest state of the default branch.](https://cli.github.com/manual/gh_release_create)